share/lutris/ui/lutris-window.ui

```diff
@@ -498,6 +468,31 @@
         <property name="border-width">9</property>
         <property name="orientation">vertical</property>
         <property name="spacing">3</property>
+        <child>
+          <object class="GtkModelButton">
+            <property name="visible">True</property>
+            <property name="can-focus">True</property>
+            <property name="receives-default">False</property>
+            <property name="action-name">win.add-game</property>
+            <property name="text" translatable="yes">Add Game</property>
```

Comment:
Can you rename it to "Add a game manually"?

Reply:
![image](https://user-images.githubusercontent.com/47753585/145651625-215bf840-efff-4a67-b579-edf17e24ae08.png)
Done. Is there anything I need to do in translation files to account for this?

Reply:
I think there might be. If I understand this, and maybe I don't, the localized text in the .po files uses the English text as a key. There are translations of "Add game" in there, and I imagine these translations will no longer be used at all.

I'm not sure what ought to be done. You might change "Add Game" in all the .po files to  "Add a game manually", so the old translations will be used. Or maybe these translations should be updated.

By the way, I think it should be "Add a Game Manually", using the same capitalization style as most of the other menu items here. ("Lutris forums" is the only one using sentence-casing  here)

Reply:
You're right, ideally it should be capitalized, the UI is a bit all over the place regarding that.

Reply:
![image](https://user-images.githubusercontent.com/47753585/145683652-1221eafc-4c70-432e-8738-2f62facd2c40.png)
Done.